src/Features/Landing/index.tsx

```diff
@@ -146,38 +154,46 @@ export const Landing = () => {
                 </video>
               </Box>
             )}
-            <Stack direction={{ base: 'column', md: 'row' }} spacing="4">
+            <Stack direction={{ base: 'column', lg: 'row' }} spacing="4">
               <Button
                 variant="simplefi-highlighted"
                 colorScheme="sip-purple-blue-gradient"
                 size={useBreakpointValue({ base: 'md', md: 'lg'})}
+                fontSize={{ md: 'lg'}}
+                isDisabled={true}
+                title="Coming soon"
                 onClick={() =>
                   window.open('https://forms.gle/pa4LonrFE63A2ygU7', '_blank')
```

Comment:
i would leave an empty action here instead of the link to the form. otherwise it can be spammed

Reply:
Button is disabled and action will need to be updated with redirection to API key process. I will delete the action in the meantime